src/mono/wasm/wasm.proj

```diff
@@ -33,7 +33,7 @@
     <_EmccDefaultsRspPath>$(NativeBinDir)src\emcc-default.rsp</_EmccDefaultsRspPath>
     <_EmccCompileRspPath>$(NativeBinDir)src\emcc-compile.rsp</_EmccCompileRspPath>
     <_EmccLinkRspPath>$(NativeBinDir)src\emcc-link.rsp</_EmccLinkRspPath>
-    <WasmNativeStrip Condition="'$(ContinuousIntegrationBuild)' == 'true'">false</WasmNativeStrip>
+    <WasmNativeStrip Condition="'$(ContinuousIntegrationBuild)' == 'true' or '$(Configuration)' == 'Debug'">false</WasmNativeStrip>
```

Comment:
`ContinuousIntegrationBuild` is suspicious here, I worry that we ship non-workload .wasm with symbols. 

@radical